### PR TITLE
Forms Automated Test - LPS-132202 | master

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by leticiachiu** [comment]
Fixed

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase [hunk 1]
```diff
@@ -193,13 +193,63 @@ definition {
...
+
+		Type(
+			key_rowNumber = "1",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 1");
+
+		Type(
+			key_rowNumber = "2",
+			locator1 = "FormFields#OPTIONS_OPTION_VALUE_FIELD",
+			value1 = "Option 2");
```
Comment (brunofariasdeo) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase:222]: Hi @leticiachiu ,
I'd like to ask you to use FormFields.editOptionsRow to edit a Single Selection option.

In this scenario, you would use something like this to edit the first option:

FormFields.editOptionsRow(			
			optionName = "Option1",			
			optionValue = "Option 1");
You can find an example here.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase [hunk 2]
```diff
@@ -193,13 +193,63 @@ definition {
...
+			fieldName = "Predefined Value",
+			fieldValue = "Option 1");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		AssertChecked.assertCheckedNotVisible(locator1 = "AppBuilder#VALIDATE_FILTER_BY_STATUS_ANY");
```
Comment (brunofariasdeo) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase:238]: I believe this assert is not required for this test scenario. 
Could you please clarify it to me? :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase [hunk 3]
```diff
@@ -193,13 +193,63 @@ definition {
...
+
+		FormViewBuilder.viewSingleSelectionChecked(
+			fieldName = "Single Selection",
+			fieldValue = "Option 1");
+
+		Click(
+			key_text = "Submit",
+			locator1 = "Button#ANY");
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
```
Comment (brunofariasdeo) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase:248]: By using FormPortlet.submitSuccessfully() you are basically trying to submit the form and expects a success message, which seems perfect in this scenario that you have designed.
May I ask you to use it here?

**Review by brunofariasdeo — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase (hunk 1 above)
Comment (leticiachiu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase:222]: Hello @brunofariasdeo, I tried to use this macro but when I use:
`FormFields.editOptionsRow(			
			optionName = "Option 1",			
			optionValue = "Option 1");

FormFields.editOptionsRow(			
			optionName = "Option 2",			
			optionValue = "Option 2");`

The first value is edited twice, but it doesn't add another field.. Do you want me to use your suggestion for Option 1 and that Type in Option 2?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase (hunk 2 above)
Comment (leticiachiu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase:238]: Ok, I gonna erase it and leave just the FormViewBuilder.viewSingleSelectionChecked part :)

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:forward

**Review by brunofariasdeo — APPROVED** [review]

**Pull request closed without merging**